MAUI/Cartesian-Charts/Legend.md

```diff
@@ -324,6 +365,23 @@ this.Content = chart;
 
 ![Legend layout for cartesian chart](Legend-images/cartesian_chart.png)
 
+## Event 
+
+**LegendItemCreated**
+
+The [`LegendItemCreated`]() event is triggered when the chart legend item is created. This argument contains the following information.
+
+* [`LegendItem`]() – Used to customize the label and appearance of individual legend item. 
+* [`TextColor`]() – used to change the color of the label.
+* [`FontFamily`]() - used to change the font family for the legend label. 
+* [`FontAttributes`]() - used to change the font style for the legend label. 
+* [`FontSize`]() - used to change the font size for the legend label.
+* [`Margin`]() - used to change the margin size of labels.
+* [`IconBrush`]() - used to change the color of the legend icon.
+* [`IconType`]() - used to change the icon type for the legend icon.
+* [`IconHeight`]() - used to change the icon height of the legend icon.
+* [`IconWidth`]() - used to change the icon width of the legend icon.
+
```

Comment:
Address these comments for all chart types legend md files

Reply:
updated in all chart types legend